rust/connlib/tunnel/src/io.rs

```diff
@@ -118,6 +118,8 @@ impl Io {
         &mut self,
         dns_servers: impl IntoIterator<Item = (IpAddr, DnsServer)>,
     ) {
+        self.forwarded_dns_queries =
```

Comment:
Easy way to replace the `forwarded_dns_queries`

Reply:
Yey for structured concurrency!